.github/workflows/makefile.yml

```diff
@@ -0,0 +1,27 @@
+name: Build RTSan Libs
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

Comment:
Is there a way to specify manual trigger? It seems like that is what we want here.

Reply:
My plan was to make this trigger on a specific version tag - but that is after we do the wrapping and publishing of the artifacts.

In the meantime you can always do manual running of actions from the "actions" tab in github